src/main/java/uk/gov/homeoffice/digital/sas/person/enums/TermsAndConditions.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.homeoffice.digital.sas.person.enums;
+
+public enum TermsAndConditions {
+  MODERNISED("MODERNISED"),
+  PRE_MODERNISED("PRE_MODERNISED");
+
+  private final String stringValue;
+
+  TermsAndConditions(final String s) {
+    this.stringValue = s;
+  }
```

Comment:
Could you use the `@AllArgsConstructor` annotation instead?

Reply:
good point, thanks